crates/dex-stable/src/lib.rs

```diff
@@ -108,6 +110,10 @@ pub mod pallet {
 
         /// Weight information for extrinsics in this pallet.
         type WeightInfo: WeightInfo;
+
+        /// Convert supported currencies to target asset.
+        /// NOTE: the price should only ever increase
```

Comment:
Is this a hard requirement?

Reply:
I think theoretically the price can also decrease if a slashing event happens (LSD) or a default without sufficient collateral to cover the debt (qToken).